tchannel/serializer/json.py

```diff
@@ -0,0 +1,30 @@
+from __future__ import absolute_import
+
+import json
+
+from tchannel.schemes import JSON
+
+
+class JsonSerializer(object):
+    def __init__(self):
+        self.name = JSON
+
+    def deserialize_header(self, obj):
+        if not obj:
+            return None
+        return json.loads(obj)
+
+    def serialize_header(self, obj):
+        if not obj:
+            obj = {}
+        return json.dumps(obj)
+
+    def deserialize_body(self, obj):
+        if not obj:
+            return None
+        return json.loads(obj)
+
+    def serialize_body(self, obj):
+        if not obj:
+            obj = {}
```

Comment:
No, I'm saying that the default of `{}` for false-like values is wrong. `call(""/False/[])` should not send `"{}"`